IconDiffBot/Core/PayloadProcessor.cs

```diff
@@ -124,17 +124,6 @@ async Task RunCheck()
 				if(allChangedFilesTask == null || allChangedFilesTask.IsFaulted)
 					allChangedFilesTask = gitHubManager.GetPullRequestChangedFiles(pullRequest, installationId, cancellationToken);
 
-				if (generalConfiguration.BlacklistedRepos.Contains(repositoryId)) {

```

Comment:
This got moved down because it kept failing due to the `checkRunId` not existing yet